View/Screens/AdminScreen.xaml

```diff
@@ -6,6 +6,7 @@
         <Grid Margin="0,0,0,0" Background="#FF303651" DataContext="{x:Static DataContexts:AdminDataContext.Instance}">
 	        <Label Content="{Binding NumberOfRequests}" ContentStringFormat="You have {0} requests" HorizontalAlignment="Center" Margin="0,42,0,0" VerticalAlignment="Top" FontSize="26.5" Foreground="White" FontWeight="Bold" Width="auto"/>
             <Button Content="Review requests" HorizontalAlignment="Center" Margin="0,111,0,0" VerticalAlignment="Top" Width="226" Height="43" FontSize="26.5"  Foreground="White" Background="#FFF78D0E" FontWeight="Bold" BorderThickness="0" Click="Review_Button_Click" />
+            <Button Content="Users" HorizontalAlignment="Center" Margin="0,180,0,0" VerticalAlignment="Top" Width="226" Height="43" FontSize="26.5"  Foreground="White" Background="#FFF78D0E" FontWeight="Bold" BorderThickness="0" Click="OpenRoleAssignmentScreen_Click" />
```

Comment:
Kan dit geen MenuButton worden? Scheelt weer een oncall functie in `AdminScreen.xaml.cs`

Reply:
Bedoel je een PermissionButton? Want het is geen MenuButton

Reply:
Hoezo is dit geen menu button? Hij refereert toch gewoon naar een nieuw scherm?

Reply:
Ja maar het staat niet in het menu

Reply:
Dat maakt echt niks uit

Reply:
Als het goed is was het de bedoeling dat dit via het AdminDashboard ging daarom is dit geen MenuButton